app/controllers/application_controller/ci_processing.rb

```diff
@@ -5,14 +5,17 @@ module ApplicationController::CiProcessing
     private(:process_elements)
   end
 
+  def ownership_form_fields
+    @ownership_items = params[:id].split(/\s*,\s*/)
+    request_hash = build_ownership_hash
+    render :json => request_hash
+  end
+
   # Set Ownership selected db records
   def set_ownership(klass = "VmOrTemplate")
     assert_privileges(params[:pressed])
-    @edit = {}
-    @edit[:key] = "ownership_edit__new"
-    @edit[:current] = {}
-    @edit[:new] = {}
-    @edit[:klass] = klass.constantize
+
+    @klass = klass.constantize
```

Comment:
Seems like you ended up using `get_class_from_controller_param(params[:controller])` instead of `@klass` so this doesn't do anything?